polaris-react/rollup.config.mjs

```diff
@@ -54,7 +54,7 @@ function generateConfig({output, targets, stylesConfig}) {
 /** @type {import('rollup').RollupOptions} */
 export default [
   generateConfig({
-    targets: 'extends @shopify/browserslist-config, node 16.13.0',
+    targets: 'node 16.13.0',
```

Comment:
Removing this seemed to be fine as it still builds with the config from `package.json`

Reply:
I expect that if you specify targets then it overrides whatever is specified in the package.json rather than complementing it - i.e. by doing this you're saying "target node only" rather than "target node and the browsers specified in the package.json.

Consider the esnext build below which has the target of `'last 1 chrome versions'`. If that merged with the contents of the browserslist key in the package.json then the main and esnext builds would be identical (as they'd both target the oldest target of ios 13.4), but that's not the case.

I think you'll want to either omit the targets entirely and add node to the browserslist key in package.json, or repeat the full browserslist here